<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Authentification</class>
 <widget class="QMainWindow" name="Authentification">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">*{
	font-family:century gothic;
	font-size:24px;
}
QFrame{
	background:#333;
	border-radius:15px;
}
#form{
	background:url(:/image/bg.jpg);
}
QPushButton{
	background:red;
	border-radius:60px;
}
QToolButton{
	background:red;
	border-radius:15px;
}
QLabel{
	color:white;
	background:transparent;
}
QPushButton{
	color:white;
	border-radius:15px;
}
QPushButton:hover{
	color:#333;
	border-radius:15px;
	background:#49ebff;
}
QLineEdit{
	background:transparent;
	border:none;
	color:#717072;
	border-bottom:1px solid #717072;
}</string>
   </property>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>100</y>
      <width>381</width>
      <height>431</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QFrame{
	background:#3
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="pageCo">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>50</y>
       <width>251</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Page de Connexion</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonSubmit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>350</y>
       <width>211</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Connexion</string>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>0</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../image/user2.png</normaloff>../image/user2.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
    </widget>
    <widget class="QLineEdit" name="userLineEdit">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>190</y>
       <width>311</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Username</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="passwordLineEdit">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>290</y>
       <width>311</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QLabel" name="username">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>150</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Username</string>
     </property>
    </widget>
    <widget class="QLabel" name="password">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>250</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
